.github/workflows/docker-aws.yaml

```
@@ -0,0 +1,136 @@
+              echo "task_definition=suite-dev-task" >> $GITHUB_OUTPUT
+              echo "tag=suite" >> $GITHUB_OUTPUT
+          else
+              echo "docker_image=$DOCKER_IMAGE_NAME:temp" >> $GITHUB_OUTPUT
+              echo "build_env=dev" >> $GITHUB_OUTPUT  
+              echo "service_name=suite-temp" >> $GITHUB_OUTPUT
+              echo "task_definition=suite-temp-task" >> $GITHUB_OUTPUT 
+              echo "tag=temp" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)